decidim-elections/app/commands/decidim/elections/trustee_zone/update_election_bulletin_board_status.rb

```diff
@@ -42,8 +42,7 @@ def fetch_election_results
               result_key = get_answer_id_from_result(key)
               answers = Decidim::Elections::Answer.where(id: result_key)
               answers.each do |answer|
-                answer.votes_count = value
-                answer.save!
+                answer.results.create(votes_count: value)
```

Comment:
⚠️ the bang version of `create` would raise an exception if an error occurs. I'd use it to avoid ignoring it.